ext/mkrf_conf.rb

```diff
@@ -27,11 +28,11 @@ def already_installed(dep)
 
   begin
     puts "Installing base gem"
-    inst = Gem::DependencyInstaller.new :prerelease => dep.prerelease?
+    inst = Gem::DependencyInstaller.new :prerelease => dep.prerelease?, :install_dir => install_dir
```

Comment:
Since we're here, could you please align code style in lines you've changed? One uses parentheses, while the other does not

Reply:
Sure, just it looks weird so I think it's better to fix such inconsistencies